### ISSUE TITLE
Use value from settings in a command

### ISSUE THREAD

**Issue by gwenzek** [markbirbeck/sublime-text-shell-command#58]
Hi thanks for this great plugin ! I really enjoy the simplicity and usability.

It would be nice to have a way to pass a string from the settings into a command.

For example now we can create a command like: "git diff {file}"

But I'd like to create a command "git log --format='{git_log_format}'" where the value of "git_log_format" would be found inside a setting file.

Can we add this feature  ?
I can do a PR if you're interested.

Basically I'll need to change "VariableSubstitution.py".

About that why do you use "create_variable_values" instead of sublime api "extract_variables()" (in [sublime.Window](https://www.sublimetext.com/docs/3/api_reference.html#sublime.Window)) ?

**Comment by markbirbeck** [comment]
Great idea.
